deploy/rotisserie.yaml

```diff
@@ -106,6 +106,7 @@ metadata:
   name: rotisserie-ingress
   annotations:
     kubernetes.io/tls-acme: "true"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
```

Comment:
I was about to make this same change, but don't use `force-ssl-redirect`, instead use `ssl-redirect`. https://github.com/kubernetes/ingress-nginx/blob/master/docs/annotations.md#tls-related